lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -116,6 +116,3 @@
     </section>
   </div>
 </section>
-
-## Modal included here so leanModal can position it outside of dashboard wrapper
-<%include file='keyword_sub_modal.html' />
```

Comment:
There's not a clean way to pass it the context variable for keywords_supported here